hybridse/src/udf/default_udf_library.cc

```diff
@@ -2679,6 +2750,58 @@ void DefaultUdfLibrary::InitUdaf() {
         )")
         .args_in<int16_t, int32_t, int64_t, float, double>();
 
+    RegisterUdafTemplate<StdUdafDef>("std")
```

Comment:
let's be compatible with spark sql: https://spark.apache.org/docs/latest/api/sql/index.html#stddev
- `std`, `stddev`, `stddev_samp` are sample standard deviation
- `stddev_pop` is population standard deviation

Reply:
ok. it's different from the `mysql` conventions.

Reply:
we're more relevant to spark, offline mode is based on spark.